packages/astro/package.json

```diff
@@ -114,12 +114,13 @@
     "build:ci": "pnpm run prebuild && astro-scripts build \"src/**/*.{ts,js}\" && pnpm run postbuild",
     "dev": "astro-scripts dev --copy-wasm --prebuild \"src/runtime/server/astro-island.ts\" --prebuild \"src/runtime/client/{idle,load,media,only,visible}.ts\" \"src/**/*.{ts,js}\"",
     "postbuild": "astro-scripts copy \"src/**/*.astro\" && astro-scripts copy \"src/**/*.wasm\"",
-    "test": "pnpm run test:node",
+    "test": "pnpm run test:node && pnpm run test:types",
     "test:match": "pnpm run test:node --match",
     "test:e2e": "pnpm test:e2e:chrome && pnpm test:e2e:firefox",
     "test:e2e:match": "playwright test -g",
     "test:e2e:chrome": "playwright test",
     "test:e2e:firefox": "playwright test --config playwright.firefox.config.js",
+    "test:types": "tsc --project tsconfig.tests.json",
     "test:node": "astro-scripts test \"test/**/*.test.js\""
```

Comment:
Can you update this so that node:test doesn't run these tests?

Reply:
Oh, do you mean exclude the `test/types` folder from what `node:test` crawls?

Reply:
Or do you mean remove the call from here? https://github.com/withastro/astro/pull/11439/files#diff-31a85c0b4c5526a896f02b675a49f1f942ec15fa815f6326edc4f74248aa0048R117

I'd like our CI to run these tests, and I thought including in `test` (but excluding in `node:test`) was the right move.